<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 94, 107);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;PATIENT RESPONSE &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="recordData">
       <property name="text">
        <string>Record Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="repeatTest">
       <property name="text">
        <string>Repeat Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;TOUCH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Force Felt</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="yesForce">
           <property name="text">
            <string>Yes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="noForce">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Finger</string>
       </property>
       <widget class="QComboBox" name="fingerSelect">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Thumb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Index</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Middle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ring</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pinky</string>
         </property>
        </item>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Comments</string>
       </property>
       <widget class="QLCDNumber" name="forceNumber">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="forceDisplay">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="begin">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No Feel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>351</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Test&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Finger&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="TestCombo">
       <item>
        <property name="text">
         <string>Force</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Two Point Discrimination</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="FingerCombo">
       <item>
        <property name="text">
         <string>Thumb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Index</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Middle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ring</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pinky</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="forceSensorPlot" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
